linuxdoc/kernel_doc.py

```diff
@@ -1854,9 +1854,9 @@ def feed(self, data, eof=False):
     def output_decl(self, name, out_type, **kwargs):
         self.ctx.offset = self.ctx.decl_offset
 
-        if name in self.translator.dumped_names:
+        if out_type + "." + name in self.translator.dumped_names:
             self.error("name '%s' used several times" % name)
```

Comment:
Can you give me a small kernel-doc example which demonstrate the effect / Thanks!